.github/workflows/code_ext.yml

```diff
@@ -0,0 +1,82 @@
+name: Code Extention CI
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - 'master'
+      - 'dev'
+
+jobs:
+  build:
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        node-version: [10.x, 12.x]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: npm install
+        run: |
+          npm ci --ignore-scripts
+          cd client
+          npm ci
+          cd ..
+          cd server
+          npm ci
+          cd ..
+        env:
+          CI: true
+
+      - name: build
+        run: |
+          npm run compile
+        env:
+          CI: true
+
+      - name: style
+        run: |
+          npm run format
```

Comment:
Remove `npm run format`.
I'll admit it could be worded better, but the `format` NPM script changes the source code while the `lint` script simply checks it.